scripts/install-chef-suse.sh

```diff
@@ -130,7 +131,7 @@ pipe_show_and_log () {
         rm -f $t
         dialog --clear >&3
     fi
-    tee -a /dev/fd/3 /dev/fd/4 > /dev/null
+    tee -a /dev/fd/4 >&3
```

Comment:
IIRC, we stole that from devstack. I'm just very reluctant to change something like this without understanding why this happens because this might mean we're missing another bug.

Did you check that this didn't break the script when installing without calling from crowbar? Also, can you reproduce the issue when manually calling the script with the same options as crowbar use?